viomem/sys/public.h

```diff
@@ -0,0 +1,30 @@
+#ifndef PUBLIC_H
+#define PUBLIC_H
+
+#include <initguid.h>
+// {E18B5FB3-04E4-42fc-9601-8395C217391B}
+DEFINE_GUID(GUID_DEVINTERFACE_BALLOON,

```

Comment:
Why these VirtIO Balloon-related definitions are here?

Reply:
viomem.vcxproj.filters is referencing public.h, but it looks that no one  is actually using it.